unified-installer/astra-unified-installer.sh

```diff
@@ -1708,16 +1935,7 @@ step_check_config() {
     done
 
     # Env vars with special conditions
-    if [ -n "$IMAGE_PULL_SECRET" ]; then
-        if [ -z "$NAMESPACE" ]; then
```

Comment:
Because `CONNECTOR_NAMESPACE` and `TRIDENT_NAMESPACE` are the source of truth and both have a default, we no longer need this check and prompt since they will never be empty.